sequence_processing_pipeline/Pipeline.py

```diff
@@ -11,6 +11,7 @@
 from sequence_processing_pipeline.PipelineError import PipelineError
 from sequence_processing_pipeline.QCJob import QCJob
 from sequence_processing_pipeline.SequenceDirectory import SequenceDirectory
+from sequence_processing_pipeline.SIF import SampleInformationFile
```

Comment:
Can you change SIF to SampleInformationFile?

Reply:
I'll remove it instead. (See comment below)